skymp5-server/cpp/addon/main.cc

```diff
@@ -316,6 +320,9 @@ ScampServer::ScampServer(const Napi::CallbackInfo& info)
     const auto& logger = GetLogger();
     partOne->AttachLogger(logger);
 
+    logger->info("Run localization provider");
+    this->localizationProvider = new LocalizationProvider("russian");
```

Comment:
Maybe this thing should be configured via config file?

Reply:
Thanks, I forgot